File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int _max = 0, count = 0;
        for (int i = 0; i < nums.size(); i++)
        {
            count = 0;
            while (i < nums.size() && nums[i] == 1)
            {
                count++;
                i++;
            }
            _max = max(_max, count);
        }
        return (_max);
    }
};

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        _max = 0;
        count = 0;
        for num in nums:
            if (num == 1):
                count = count + 1;
            else:
                _max = max(_max, count);
                count = 0;
        _max = max(_max, count);
        return (_max);